Formula/orientdb.rb

```diff
@@ -48,6 +43,16 @@ def post_install
     (var/"log/orientdb").mkpath
     touch "#{var}/log/orientdb/orientdb.err"
     touch "#{var}/log/orientdb/orientdb.log"
+
+    # set the server's root user  password to changeme
+    ENV["ORIENTDB_ROOT_PASSWORD"] = "changeme"
```

Comment:
Sorry, I don't know what a `caveats` block is. I'm not an expert of brew, I read the cookbook: :https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md#formula-cookbook